src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java

```diff
@@ -426,4 +426,12 @@
         assertFalse(p.allowsScriptFromSource(new GUID("data:")));
     }
 
+    @Test public void testHasSomeEffect() {
+        Policy p = Parser.parse("script-src *", "http://example.com");
```

Comment:
I would prefer just the empty policy here. Remember, we're changing this. `script-src *` doesn't allow inline scripts.